<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgConnHidWifiSecTls</class>
 <widget class="QDialog" name="DlgConnHidWifiSecTls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connect to Hidden Wi-Fi Network</string>
  </property>
  <widget class="QComboBox" name="cbxSecurity">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>165</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox QAbstractItemView {
	outline: 0; 
	border: 1px solid #000000;
	border-color: rgba(0, 0, 0,25%);
	selection-color: rgb(0, 0, 0);
	background-color: #F1F3F3;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNetName">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>125</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="leUserCertifyPwd">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>485</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="leCaPwd">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>365</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxPwd">
   <property name="geometry">
    <rect>
     <x>357</x>
     <y>377</y>
     <width>18</width>
     <height>9</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbConn">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>60</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbNetName">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>130</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbDomain">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>290</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbCA">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>330</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxUserCertify">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>445</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox QAbstractItemView {
	outline: 0; 
	border: 1px solid #000000;
	border-color: rgba(0, 0, 0,25%);
	selection-color: rgb(0, 0, 0);
	background-color: #F1F3F3;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbIdentity">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>250</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbUserCertifyPwd">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>490</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>634</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbUserKeyPwd">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>570</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxCA">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>325</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox QAbstractItemView {
	outline: 0; 
	border: 1px solid #000000;
	border-color: rgba(0, 0, 0,25%);
	selection-color: rgb(0, 0, 0);
	background-color: #F1F3F3;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbUserPriKey">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>530</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbCaPwd">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>370</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="leDomain">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>285</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxPwdSec">
   <property name="geometry">
    <rect>
     <x>337</x>
     <y>577</y>
     <width>18</width>
     <height>8</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="leIdentity">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>245</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="leUserKeyPwd">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>565</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbUserCertify">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>450</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbAuth">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>210</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxAuth">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>205</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox QAbstractItemView {
	outline: 0; 
	border: 1px solid #000000;
	border-color: rgba(0, 0, 0,25%);
	selection-color: rgb(0, 0, 0);
	background-color: #F1F3F3;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConnect">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>634</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxCA">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>405</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxConn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>55</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox QAbstractItemView {
	outline: 0; 
	border: 1px solid #000000;
	border-color: rgba(0, 0, 0,25%);
	selection-color: rgb(0, 0, 0);
	background-color: #F1F3F3;
}</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxUserPriKey">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>525</y>
     <width>200</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox QAbstractItemView {
	outline: 0; 
	border: 1px solid #000000;
	border-color: rgba(0, 0, 0,25%);
	selection-color: rgb(0, 0, 0);
	background-color: #F1F3F3;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbSecurity">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>170</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbBoder">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>432</width>
     <height>680</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbUpback">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>432</width>
     <height>116</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbBottomback">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>620</y>
     <width>432</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbLeftup">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbLeftupIcon">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbLeftupTitle">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>6</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>lbBottomback</zorder>
  <zorder>lbUpback</zorder>
  <zorder>lbBoder</zorder>
  <zorder>cbxSecurity</zorder>
  <zorder>leNetName</zorder>
  <zorder>leUserCertifyPwd</zorder>
  <zorder>leCaPwd</zorder>
  <zorder>checkBoxPwd</zorder>
  <zorder>lbConn</zorder>
  <zorder>lbNetName</zorder>
  <zorder>lbDomain</zorder>
  <zorder>lbCA</zorder>
  <zorder>cbxUserCertify</zorder>
  <zorder>lbIdentity</zorder>
  <zorder>lbUserCertifyPwd</zorder>
  <zorder>btnCancel</zorder>
  <zorder>lbUserKeyPwd</zorder>
  <zorder>cbxCA</zorder>
  <zorder>lbUserPriKey</zorder>
  <zorder>lbCaPwd</zorder>
  <zorder>leDomain</zorder>
  <zorder>leIdentity</zorder>
  <zorder>leUserKeyPwd</zorder>
  <zorder>lbUserCertify</zorder>
  <zorder>lbAuth</zorder>
  <zorder>cbxAuth</zorder>
  <zorder>btnConnect</zorder>
  <zorder>checkBoxCA</zorder>
  <zorder>cbxConn</zorder>
  <zorder>cbxUserPriKey</zorder>
  <zorder>lbSecurity</zorder>
  <zorder>lbLeftup</zorder>
  <zorder>lbLeftupIcon</zorder>
  <zorder>lbLeftupTitle</zorder>
  <zorder>checkBoxPwdSec</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
